build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group = "com.emergetools"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
I think it should be 3.0.1, according to semantic versioning, this sounds like a patch